content/millennium/r4/foundation/security/provenance.md

```diff
@@ -9,31 +9,37 @@ title: Provenance | R4 API
 
 ## Overview
 
-The Provenance resource tracks the source of external data and is used to assess the trustworthiness and reliability of a resource. Sources can be an organization author or organization transmitter.  Entities are Continuity of Care DocumentReferences (CCDs).
+The Provenance resource tracks the source of external information and is used to assess the trustworthiness and reliability of a resource. Sources can be an organization author or organization transmitter. The target references a resource and ID that this Provenance entry is related to. Entities are Continuity of Care Documents (CCDs) generated through the DocumentReference resource.
```

Comment:
"The target" -> "The `target` field"

"resource" is generally acceptable as a reference to a specific entity (Patient/1234), so probably don't need to include "and ID"

granted, most of this is just repetitive of the HL7 spec definitions :shrug:

Reply:
I liked these changes too - adjusted.